src/OrchardCore.Modules/OrchardCore.PublishLater/Views/PublishLaterPart.Edit.cshtml

```diff
@@ -0,0 +1,18 @@
+@using Microsoft.AspNetCore.Authorization
+@model PublishLaterPartViewModel
+
+@inject IAuthorizationService AuthorizationService
+
+@if (await AuthorizationService.AuthorizeAsync(User, OrchardCore.Contents.Permissions.PublishContent, Model.ContentItem))
+{
+    <div class="btn-group">
+        <input asp-for="ScheduledPublishLocalDateTime" type="datetime-local" class="form-control content-preview-select" />
+        <button class="btn btn-primary" type="submit" name="submit.Save" value="submit.PublishLater">@T["Publish&nbsp;Later"]</button>
```

Comment:
I'm not sure why we can't use `@T["Publish Later"]` instead?!! I don't think we need to use `&nbsp;`

Reply:
There's a nbsp here and in the cancel button because otherwise the buttons' texts can be wrapped on certain screen sizes. E.g. this is with nbsp:

![image](https://user-images.githubusercontent.com/1976647/79048177-870da100-7c1b-11ea-82d8-fb7676aadbb0.png)

This is without:
![image](https://user-images.githubusercontent.com/1976647/79048193-98ef4400-7c1b-11ea-8f13-7e7d15a83b08.png)

Probably a matter of taste but I like the first one better.

Reply:
Hmm but now that I think about it this would indeed be nicer from CSS, you're right. I'll change this.

Reply:
At least it proves that when using an `IViewLocalizer` the main string is not html encoded ;) Only `{0}` parameters are encoded, this allows to include html tags, an entity as `&nbsp;` ...

So the main strings of our translations using `T[...]` may need to be html pre-encoded, not sure we are aware of this ;)

Reply:
Changed this and the cancel button to do the same from CSS.